tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -147,7 +147,7 @@ spec:
         fi
 
         echo 'Initializing TUF root...'
-        cosign initialize --mirror "${TUF_MIRROR}" --root "${TUF_MIRROR}/root.json"
+        ec sigstore initialize --mirror "${TUF_MIRROR}" --root "${TUF_MIRROR}/root.json"
```

Comment:
Just brainstorming, but we could consider calling it `ec sigstore cosign-initialize`  - it would leave the command namespace more open in case we want to add other sigstore stuff and make it more clear what this is equivalent to.

Reply:
So the initialize also works for `gitsign`. I think `sigstore` is the right level.